guides/getting-started.md

```diff
@@ -49,6 +49,7 @@ import {NoopAnimationsModule} from '@angular/platform-browser/animations';
 })
 export class PizzaPartyAppModule { }
 ```
+If you are using Angular Universal the animation modules need to be imported in your browser specific code path such as browser.module.ts, not the main app.module.ts
```

Comment:
I would combine this with the existing blurb about `NoopAnimationsModule`

The wording would be something like
```markdown
Note that this is the _browser_ animations module. If you are targeting `@angular/platform-server`,
or if you simply want to disable all animations, you can instead use `NoopAnimationsModule`.
```
`@angular/platform-server`,